File: massa_de_dados/GeradorDados/Models/TipoEmpresa.cs
namespace GeradorDados.Models;

public class TipoEmpresa
{
    public TipoEmpresa(int tipoempresaid, string nome)
    {
        Tipoempresaid = tipoempresaid;
        Nome = nome;
    }

    public int Tipoempresaid { get; set; }
    public string Nome { get; set; }
}

File: api/src/main/java/br/ucsal/clinica/model/TipoEmpresa.java
package br.ucsal.clinica.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


@Entity
@Table(name = "tipo_empresa")
@AllArgsConstructor
@NoArgsConstructor
@Data
public class TipoEmpresa {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String nome;
}
